docs-website/src/pages/_components/TownhallButton/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from 'react';
+import styles from "./townhallbutton.module.scss";
+import clsx from "clsx";
+import Link from "@docusaurus/Link";
+
+const TownhallButton = () => {
+  const [showButton, setShowButton] = useState(false);
+  const [currentMonth, setCurrentMonth] = useState('');
+
+  useEffect(() => {
```

Comment:
none of this needs to be in a useEffect - this component shouldn't need any state